### ISSUE TITLE
Incorrect overflowing text

### ISSUE THREAD

**Issue by orzogc** [ronjb/float_column#9]
Example code:
```dart
import 'package:flutter/material.dart';
import 'package:float_column/float_column.dart';

const String text =
    '''Once upon a time, in a small town located in the heart of America, there was a young girl named Lily.
She was the daughter of a famous and wealthy businessman, and had everything that anyone could ever want.
However, despite her privileged upbringing, Lily was secretly unhappy and longed for something more.
One day, while browsing in a local bookstore, she came across a dusty old book that caught her eye.
It was a collection of short stories written by an acclaimed author, and Lily knew that she had to have it.
As she reached for the book, she noticed a young man standing beside her, also looking at the same book.
He was handsome and charming, with a mischievous glint in his eye.
"Looks like we both have a good taste in literature," he said, flashing a smile at her.
Lily felt her heart skip a beat. She had never experienced such strong feelings before.
They began to talk and discovered they shared a love of books and a deep sense of adventure.
They decided to go on a spontaneous road trip through the country, exploring new sights and creating incredible memories.
''';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      debugShowCheckedModeBanner: false,
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatelessWidget {
  const MyHomePage({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: FloatColumn(
        children: [
          Floatable(
              float: FCFloat.start,
              padding: const EdgeInsets.only(right: 5.0),
              child: Image.network(
                  'https://flutter.github.io/assets-for-api-docs/assets/widgets/rich_text.png')),
          RichText(
            text: const TextSpan(text: text),
            maxLines: 8,
            overflow: TextOverflow.ellipsis,
          ),
        ],
      ),
    );
  }
}
```

float_column 1.3.3:
![Screenshot_20230512_192500](https://github.com/ronjb/float_column/assets/6040599/bf007365-f6e0-4a96-8ad1-d1753993a5e5)

float_column 2.0.0:
![Screenshot_20230512_192641](https://github.com/ronjb/float_column/assets/6040599/686a53e2-46ce-4aeb-a701-80364e87946f)

**Comment by ronjb** [comment]
@orzogc, thanks for reporting this bug. It is now fixed in version 2.0.1. Let me know if you still see problems.

**Comment by orzogc** [comment]
It was fixed, thanks!
